roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -76,6 +76,7 @@
       osd: allow *
       mds: allow *
     cluster: "{{ cluster }}"
+    secret: "{{ (mgr_secret is defined and mgr_secret != 'mgr_secret') | ternary(mgr_secret, omit) }}"
```

Comment:
`mgr_secret is defined` is not necessary since it's declared in `defaults/main.yml`